src/qt/res/src/network_disabled.svg

```diff
@@ -5,11 +5,11 @@
    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
    xmlns:svg="http://www.w3.org/2000/svg"
    xmlns="http://www.w3.org/2000/svg"
-   id="svg2"
-   viewBox="0 0 24 24"
-   height="24"
+   version="1.2"
    width="24"
-   version="1.2">
+   height="24"
```

Comment:
This is not required. You can tell the program of your choice what dpi to sample at.

Nonetheless, I changed it to `92`, as well as for all `connect-$N.svg`